### ISSUE TITLE
GitHub Action: thermal limit

### ISSUE THREAD

**Issue by timlichtenberg** [FormingWorlds/PROTEUS#51]
- Introduce unit test for AEOLUS to ensure thermal limit in pure steam atmosphere is always reproduced

**Comment by timlichtenberg** [comment]
@nichollsh This is already implemented partially via an executable script, right? I presume we 'only' need to automate its execution now? Which script is this?

**Comment by nichollsh** [comment]
Yeah, there's a script which calculates OLR for a range of Tsurf values in order to reproduce the transition in/out of the runaway state. 
https://github.com/FormingWorlds/JANUS/blob/master/demo_runaway_greenhouse.py

**Comment by timlichtenberg** [comment]
Ok, so what do we think should an automated test look like for an initial try of this? I presume it should run this script for the pure steam case and check out the output within a fixed temperature range to reproduce the ~value within a range +/-5 K? One may consider also checking for the inflection point at high T, but the sensitivity on the adopted opacity makes this a bit more complicated I guess.

**Comment by nichollsh** [comment]
I think that checking the fluxes at a fixed point makes the most sense. This would make the test quicker to perform, and will catch just as many issues. Maybe evaluating the OLR at Tsurf=1700 K, or something like that?

If it helps, I've already made [a test script for AGNI](https://github.com/nichollsh/AGNI/blob/main/demo_tests.jl) which includes a check for the runaway OLR limit. We could adopt something similar. I don't know much about how to implement unit tests, but it seems that delegating them to a single script and having it return pass/fai
